.editorconfig

```diff
@@ -3,8 +3,17 @@ root = true
 
 ; Unix-style newlines
 [*]
+charset = utf-8
 end_of_line = LF
+insert_final_newline = true
+trim_trailing_whitespace = true
 
 [*.php]
 indent_style = space
 indent_size = 4
```

Comment:
I'm wondering, do we have any files with an `indent_size` of 2, or that needs tabs? Otherwise we could move this to `[*]` in my opinion.

Reply:
*.js, *.md, *.sass, *.css may have different  indent

Reply:
`Makefile` is an example where tabs would be needed. However, there are no makefiles in this repository, so it wouldn't be an issue.